pkg/scheduler/scheduler.go

```diff
@@ -607,13 +607,18 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 	}()
 }
 
+var attempts = []string{"0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15"}
+
 func getAttemptsLabel(p *framework.QueuedPodInfo) string {
 	// We breakdown the pod scheduling duration by attempts capped to a limit
 	// to avoid ending up with a high cardinality metric.
 	if p.Attempts >= 15 {
 		return "15+"
 	}
-	return string(p.Attempts)
+	if p.Attempts < 0 {
+		return "-"
+	}
+	return attempts[p.Attempts]
```

Comment:
cc @ahg-g 

avoided strconv here since it is expensive for frequently called paths

Reply:
this is not called frequently, once per scheduled pod.

Reply:
FWIW I don't think this codepath is executed that frequently; it's executed once per successfully scheduled pod, so IMO I prefer the simpler approach.

(Also, `Itoa` has a fast path for small ints: https://golang.org/src/strconv/itoa.go?s=784:824#L15)

Reply:
> `Itoa` has a fast path for small ints: https://golang.org/src/strconv/itoa.go?s=784:824#L15

ah, great... I cargo-culted this wisdom from our request metrics which log the response code which can be up in the 5xx range and exceeds the small number optimization

simplified back to strconv here